app/scripts/views/mixins/flow-begin-mixin.js

```diff
@@ -10,10 +10,7 @@ define(function (require, exports, module) {
 
   module.exports = {
     afterRender () {
-      const flowId = this.flow.get('flowId');
-      const flowBegin = this.flow.get('flowBegin');
-
-      this.metrics.logFlowBegin(flowId, flowBegin);
+      this.logFlowEventOnce('begin');
```

Comment:
Although also, I realise now, that navigation argument becomes moot with this change. But the resume token one doesn't, we still need it for that.